src/components/sandbox/Sandbox.tsx

```diff
@@ -104,60 +105,61 @@ export const Sandbox = (props: ElementProps & { children: ReactNode }) => {
 
   function render() {
     if (lang === "angular" && props.flags?.includes("reactive")) {
-      return <>
-        Angular
-        {getCodeSnippets("angular")}
-        <CodeSnippet lang={formatLang} allowCopy={true}>
-          {output(serializers["angular"])}
-        </CodeSnippet>
-
-        Reactive
-        {getCodeSnippets("angular", "reactive")}
-        <CodeSnippet lang={formatLang} allowCopy={true}>
-          {output(serializers["angular-reactive"])}
-        </CodeSnippet>
-      </>
+      return (
+        <>
+          <h4>Event based</h4>
+          {getCodeSnippets("angular")}
+          <CodeSnippet lang={formatLang} allowCopy={true}>
+            {output(serializers["angular"])}
+          </CodeSnippet>
+          <h4>Reactive forms (FormControl)</h4>
```

Comment:
This doesn't have to be changed now, but for readability try to add line breaks in between related blocks of code/html. `h` tags are a a pretty good sign of a block of code similar to a paragraph in a book.